tracim/tests/functional/test_session.py

```diff
@@ -1,57 +1,73 @@
 # coding=utf-8
-import pytest
-
 from tracim.tests import FunctionalTest
+from tracim.tests import FunctionalTestNoDB
 
 
 class TestLogoutEndpoint(FunctionalTest):
 
-    def test_logout(self):
+    def test_api__access_logout_get_enpoint__ok__nominal_case(self):
+        res = self.testapp.post_json('/api/v2/sessions/logout', status=204)
+
+    def test_api__access_logout_post_enpoint__ok__nominal_case(self):
         res = self.testapp.get('/api/v2/sessions/logout', status=204)
 
 
+class TestLoginEndpointUnititedDB(FunctionalTestNoDB):
+
+    def test_api__try_login_enpoint__err_500__no_inited_db(self):
+        params = {
+            'email': 'admin@admin.admin',
+            'password': 'admin@admin.admin',
+        }
+        res = self.testapp.post_json(
+            '/api/v2/sessions/login',
+            params=params,
+            status=500,
+        )
+
+
 class TestLoginEndpoint(FunctionalTest):
 
-    def test_login_ok(self):
+    def test_api__try_login_enpoint__ok_204__nominal_case(self):
         params = {
             'email': 'admin@admin.admin',
             'password': 'admin@admin.admin',
         }
-        res = self.testapp.get(
+        res = self.testapp.post_json(
             '/api/v2/sessions/login',
+            params=params,
             status=204,
-            params=params
         )
 
-    def test_bad_password(self):
+    def test_api__try_login_enpoint__err_400__bad_password(self):
         params = {
             'email': 'admin@admin.admin',
             'password': 'bad_password',
         }
-        res = self.testapp.get(
+        res = self.testapp.post_json(
             '/api/v2/sessions/login',
             status=400,
             params=params,
         )
 
-    def test_bad_user(self):
+    def test_api__try_login_enpoint__err_400__unregistered_user(self):
         params = {
             'email': 'unknown_user@unknown.unknown',
             'password': 'bad_password',
         }
-        res = self.testapp.get(
+        res = self.testapp.post_json(
             '/api/v2/sessions/login',
             status=400,
             params=params,
         )
 
-    def test_uncomplete(self):
-        res = self.testapp.get('/api/v2/sessions/login', status=400)
+    def test_api__try_login_enpoint__err_400__no_json_body(self):
+        res = self.testapp.post_json('/api/v2/sessions/login', status=400)
 
 
 class TestWhoamiEndpoint(FunctionalTest):
 
-    def test_whoami_ok(self):
+    def test_api__try_whoami_enpoint__ok_200__nominal_case(self):
```

Comment:
add asserts about expected content. You should test that name / email address are those you expect

Reply:
that's already done.